src/app/shell/personal-cabinet/provider/create-provider-admin/create-provider-admin.component.ts

```diff
@@ -121,12 +122,22 @@ export class CreateProviderAdminComponent extends CreateFormComponent implements
   }
 
   setEditMode(): void {
+    combineLatest([this.providerAdmin$, this.truncatedItems$])
+      .pipe(
+        filter(([providerAdmin, allEntities]: [ProviderAdmin, TruncatedItem[]]) => !!(providerAdmin && allEntities)),
+        takeUntil(this.destroy$)
+      )
+      .subscribe(([providerAdmin, allEntities]) => {
+        this.ProviderAdminFormGroup.patchValue(providerAdmin, { emitEvent: false });
+        this.entityControl.setValue(
+          allEntities.filter((entity: TruncatedItem) => {
+            const isChecked = !!providerAdmin.workshopTitles.filter((checkedEntity) => entity.id === checkedEntity.id).length;
```

Comment:
`filter` can be replaced by `find`.
we can remove `isChecked` variable and just return value like `() => items.find(...)`